README.md

```diff
@@ -67,14 +67,22 @@ If you followed the previous steps you should already be inside your local clone
     ![View Details Location](doc/source/images/workspace_details.png?raw=true)
 9. In the Unity editor project tab, select Assets->Scenes->MainGame->MainMenu and double click to load the scene.
 10. Press Play
-11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Cntrl + Shift +B) and click Build. When prompted you can name your build and then move it to your phone. 
-Alternately, connect the phone via USB and File-> Build and Run (or Control+B).
-    Make sure you have enabled USB Debugging:
-    A. Open Settings-> About-> Software Information-> More.
-    B. Then tap �Build number� seven times to enable Developer options. ...
-    C. Go back to Settings menu and now you'll be able to see �Developer options� there.
-    D. Tap it and turn on USB Debugging from the menu on the next screen.
+11. To Build an android .apk file and deploy to your phone, you can File -> Build Settings (Ctrl + Shift +B) and click Build. 
+12. When prompted you can name your build and then move it to your phone.
+13. Alternately, connect the phone via USB and File-> Build and Run (or Ctrl+B).
+
+   *Make sure you have enabled USB Debugging:*
```

Comment:
this block looks OK, but it is a little strange like a code block.  bullets or sub-list might look better